### PR TITLE
update documentation, add multi-column relationship

### PR DESCRIPTION
**Pull request by thathert**
This updates the DATABASE readme to be more accurate and makes two changes:

- adds createdDate to study and assets
- adds a new table, study_relationship_column, that allows us to define relationships between multiple columns

### CODE REVIEW

File: src/main/resources/db/changesets/20190117_createddates_and_multicolumn.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        - dropColumn:
+            tableName: study_relationship
+            columnName: from_column
+
+        - dropColumn:
+            tableName: study_relationship
+            columnName: to_column
+
+        - createTable:
```
Comment (ddietterich) [inline src/main/resources/db/changesets/20190117_createddates_and_multicolumn.yaml]: I think you will need some sequence number column in here. The order of the columns is significant:
(id, timestamp) is different from (timestamp, id).

Alternately, you could use an array in the study_relationship table and skip this table.

Comment (thathert) [reply src/main/resources/db/changesets/20190117_createddates_and_multicolumn.yaml]: I toyed around with using an array but as far as I can tell there is no way to define a foreign key on items in the array. I'd prefer to get the referential integrity guarantees we get from the FKs so I added a sequence_num column instead.

**Review by RoriCremer — APPROVED** [review]

File: src/main/resources/db/changesets/20190117_createddates_and_multicolumn.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                  type: timestamptz
+                  defaultValueComputed: now()
+                  constraints:
+                    nullable: false
+
+        - addColumn:
+            tableName: asset_specification
+            columns:
+            - column:
+                name: created_date
```
Comment (ahaessly) [inline src/main/resources/db/changesets/20190117_createddates_and_multicolumn.yaml]: isn't this the same created date as study?

File: src/main/resources/db/changesets/20190117_createddates_and_multicolumn.yaml (hunk 1 above)
Comment (ahaessly) [reply src/main/resources/db/changesets/20190117_createddates_and_multicolumn.yaml]: (i know we talked, but commenting here also). i thought we had decided we didn't need to support multicolumn relationships initially

**Review by ahaessly — APPROVED** [review]

**Pull request merged**